src/index.html

```diff
@@ -19,9 +19,389 @@
-    <h1>Catalog</h1>
+  <body id="home">
+    <header class="header">
+      <a
+        class="header__logo-link"
+        href="#home"
+      >
+        <img
+          class="image-logo"
+          src="./images/logo.png"
+          alt="logo"
+        />
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link is-active"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              Samsung
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              Smartphones
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              Gadgets
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              Tablets
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              Photo
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="main">
+      <article
+        class="card"
+        data-qa="card"
+      >
+        <img
+          src="images/imac.jpeg"
+          alt="imac image"
+          class="card__image"
+        />
+        <p class="card__product-name">
+          APPLE A1419 iMac 27" Retina
+          <br />
+          5K Monoblock (MNED2UA/A)
+        </p>
+        <p class="card__product-code">Product code: 195434</p>
+
+        <div class="reviews">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <p class="reviews__title">Reviews: 5</p>
+        </div>
+
+        <div class="cost">
+          <p class="cost__name">Price:</p>
+          <p class="cost__value">$2,199</p>
+        </div>
+
+        <a
+          href="#"
+          class="button"
+          data-qa="card-hover"
+        >
+          Buy
+        </a>
+      </article>
+      <article class="card">
```

Comment:
Add the blank line between the cards
```suggestion
      </article>
      
      <article class="card">
```